sdk/include/psdk/shlobj.h

```diff
@@ -336,6 +336,10 @@ SHOpenFolderAndSelectItems(
 
 int WINAPI PathCleanupSpec(_In_opt_ LPCWSTR, _Inout_ LPWSTR);
 
+#if (_WIN32_WINNT_WINXP <= _WIN32_WINNT) && (_WIN32_WINNT <= _WIN32_WINNT_WS03)
```

Comment:
```suggestion
#if (_WIN32_WINNT >= _WIN32_WINNT_WINXP) && (_WIN32_WINNT <= _WIN32_WINNT_WS03)
```

Reply:
lol this is what I originally suggested ^^